analyses/focal-cn-file-preparation/rna-expression-validation.R

```diff
@@ -116,7 +116,7 @@ source(
 ambiguous_sample_ids <- metadata %>%
   dplyr::filter(
     sample_type == "Tumor",
-    composition == "Solid Tissue"
+    composition == "Solid Tissue" | composition == "Bone Marrow"
```

Comment:
Ahh - can you make a ticket to update this later @runjin326 (since it does not affect the v7 output) - but we should remove this totally - this was done to not capture cell lines, but we should capture all of these in our output files and filter downstream later.